packages/legend-application-data-cube/src/stores/builder/LegendDataCubeBuilderStore.tsx

```diff
@@ -295,6 +299,27 @@ export class LegendDataCubeBuilderStore {
         const specification = DataCubeSpecification.serialization.fromJson(
           persistentDataCube.content,
         );
+
+        if (
+          !this.saveState.hasSucceeded &&
```

Comment:
why we use `saveState` here??

Reply:
cause we loadDataCube after we save too, so for local source this would ask user to upload source which we don't technically require